istore--0.1.1--0.1.2.sql

```diff
@@ -1,38 +1,107 @@
 ----functions----
+CREATE FUNCTION istore_agg_finalfn_pairs(internal)
+ RETURNS istore
+ LANGUAGE c
+ IMMUTABLE STRICT
+AS 'istore', $function$istore_agg_finalfn_pairs$function$;
+
+CREATE OR REPLACE FUNCTION bigistore_agg_finalfn(internal)
+ RETURNS bigistore
+ LANGUAGE c
+ IMMUTABLE STRICT
+AS 'istore', $function$bigistore_agg_finalfn_pairs$function$;
+
+CREATE FUNCTION istore_max_transfn(internal, bigistore)
+ RETURNS internal
+ LANGUAGE c
+ IMMUTABLE
+AS 'istore', $function$bigistore_max_transfn$function$;
+
+----
+CREATE FUNCTION istore_min_transfn(internal, bigistore)
+ RETURNS internal
+ LANGUAGE c
+ IMMUTABLE
```

Comment:
@zeitgeist I had the same train of thought earlier, but the because the types are different we can't use the strict definition.